### PR TITLE
eventLimitRender option

### PR DESCRIPTION
**Pull request by rumbu**
Added an option `eventLimitRender` to have an ability to manage the 
rendering of "more" links like it is for events using `eventRender` 
option. Docs will be in the branch docs-2.0, separately. If you have 
any suggestions, please, comment.

### CODE REVIEW

**Comment by ddalcu** [comment]
cool, this is just what I was looking for !

**Comment by rumbu** [comment]
An example view:
Example screenshot

**Comment by arshaw** [comment]
thanks for this. and thanks for the docs (in #2833)

File: dist/fullcalendar.js [hunk 1]
```diff
@@ -5728,6 +5728,12 @@ DayGrid.mixin({
 				else if (typeof clickOption === 'string') { // a view name
 					view.calendar.zoomTo(date, clickOption);
 				}
+			})
+			.each(function() {
```
Comment (arshaw) [inline dist/fullcalendar.js]: no need to do an `each` for just one element here, which you then have to do `$(this)` for. can you use the `$("<a>")` previously created, directly?

File: dist/fullcalendar.js [hunk 2]
```diff
@@ -5728,6 +5728,12 @@ DayGrid.mixin({
...
+				var renderCallback = view.opt('eventLimitRender');
```
Comment (arshaw) [inline dist/fullcalendar.js]: could you please use `View::trigger` to call the handler(s)?

**Comment by arshaw** [comment]
I'd prefer an API more similar to [eventLimitClick](https://github.com/rumbu/fullcalendar/blob/b5a860c01a567878f6464a38c92287c9ea5f64c1/src/common/DayGrid.limit.js#L207-L213) where an object with named arguments is passed to the handlers.

**Comment by arshaw** [comment]
Could resubmit this PR without committing the `dist` files? only changes in `src`

**Comment by rumbu** [comment]
Committed the changes based on @arshaw suggestions.

**Comment by arshaw** [comment]
code looks great!

the last piece of the puzzle is adding some simple automated tests to prevent regressions in the future
https://github.com/fullcalendar/fullcalendar/wiki/Automated-Tests
https://github.com/fullcalendar/fullcalendar/tree/master/tests/automated
^
writing tests can sometimes be confusing, so lemme know if you need help

**Comment by arshaw** [comment]
we now have these hooks in v5: https://fullcalendar.io/docs/more-link-render-hooks

**Pull request closed without merging**